src/js/components/awardv2/visualizations/amounts/AggregatedAwardAmounts.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * AggregatedAwardAmounts.jsx
+ * Created by David Trinh 2/8/19
+ **/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChartLoadingMessage from 'components/search/visualizations/ChartLoadingMessage';
+import ChartError from 'components/search/visualizations/ChartError';
+import AggregatedAwardAmountsInfo from 'components/awardv2/visualizations/amounts/AggregatedAwardAmountsInfo';
+
+const propTypes = {
+    awardAmounts: PropTypes.object,
+    loading: PropTypes.bool,
+    error: PropTypes.bool
+};
+
+export default class AggregatedAwardAmounts extends React.Component {
+    render() {
+        let data = null;
+        if (this.props.loading) {
+            // API request is still pending
+            data = (<ChartLoadingMessage />);
```

Comment:
It looks like you need to import the styling for the loading message:
<img width="733" alt="screen shot 2019-02-25 at 2 51 27 pm" src="https://user-images.githubusercontent.com/7108785/53364480-dfe18380-390c-11e9-9883-6990f45d0416.png">